pandapower/converter/cim/cim_tools.py

```diff
@@ -72,7 +72,7 @@ def extend_pp_net_cim(net: pandapowerNet, override: bool = True) -> pandapowerNe
     fill_dict['storage'] = dict()
     fill_dict['storage'][np_str_type] = [sc['t'], 'description']
     fill_dict['shunt'] = dict()
-    fill_dict['shunt'][np_str_type] = [sc['t'], 'description']
+    fill_dict['shunt'][np_str_type] = [sc['t'], 'description', 'id_characteristic_table', 'step_dependency_table']
```

Comment:
should happen in create_empty_network, because its affecting not only CGMES converted networks